examples/ChannelsController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Channel;
+use App\User;
+use Cmgmyr\Messenger\Models\Message;
+use Cmgmyr\Messenger\Models\Thread;
+use Illuminate\Support\Facades\Auth;
+
+class ChannelsController extends MessagesController
```

Comment:
This controller does the same like the threads one - it would be better to just change the order or style of the public threads in the normal/old MessagesController.

Reply:
but this style gets a thread based on an object, not from knowing a thread.
In real usage the `Channel::first()` would be more like `Channel::where('id'=>$id);`
or if you like... `Car::where('id'=>$id);` as well as `Fruit::where('name'=>'apple');`

Reply:
Yeah, but if you want to use this like a comment system I recommend a comment package. If this should be a thread with a specific subject it should be in the normal thread list but with a special icon, label, color or something like this.
Like facebook - all threads are in the same list. Independent of if it's a 1:1, group, page or bot-chat.

Reply:
hmm yeah I guess just using a comment package instead could do it... guess I was just thinking it'd be nice to have private messages and public chats driven by the same system.